.github/workflows/build_hub.yml

```diff
@@ -232,3 +186,50 @@ jobs:
         if: ${{ steps.check_changes.outcome == 'failure' }}
         run: |
           false
+
+  docker_build_test:
+    name: Testing Docker Build
+    runs-on: ubuntu-latest
+    needs: pre_job
+    if: ${{ needs.pre_job.outputs.has_router_change == 'true' }}
+    defaults:
+      run:
+        working-directory: prime-router
+
+    steps:
+      - name: "Check out changes"
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: 'gradle'
+
+      - uses: gradle/wrapper-validation-action@v1
+
+      - name: Spin up build containers
+        run: docker-compose -f docker-compose.postgres.yml up -d
+
+      - name: Build Prime Router Package
+        run: ./gradlew package -x fatjar -x test
+
+      - name: Build Docker Image
+        run: docker build . --file Dockerfile --tag cdcgov/reportstream:latest
+          
+  linting:
+    name: Check Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deprecated step always succeeds
+        run: true
+        
+  docs:
+    # Deprecated job.  Check schema docs was moved to build_router job
```

Comment:
Context: Will need to remove this once the check in Github is removed and everyone merge all these changes from master.